### DISCUSSION TITLE
How to make input text visible in Shell: panel?

### DISCUSSION THREAD

**Discussion by knodalyte** [sxyazi/yazi#568] [1. Q&A]
When I want to enter a shell command, the text of the command does not show up in the input area, although if I select it with my mouse, I can see that it is there. 
What I think is the relevant area from my theme.toml file is this:
```
[input]
border   = { fg = "blue" }
title    = { fg = "lightgreen"}
value    = { fg = "lightyellow"}
selected = { reversed = true }
```
What am I doing wrong?

**Comment by sxyazi** [comment]
Should be fixed in Yazi v0.2.2, https://github.com/sxyazi/yazi/releases/tag/v0.2.2
